rdrf/rdrf/apps.py

```diff
@@ -12,8 +12,6 @@ class RDRFConfig(AppConfig):
 
     def ready(self):
         logger.info("running RDRFConfig.ready ... ")
-        import rdrf.account_handling.backends
```

Comment:
Should I put these back in? not really sure what they're there for...

Reply:
Yeah I think so, I believe they are setting up [signals](https://docs.djangoproject.com/en/dev/topics/signals/)

Reply:
Done.